.github/workflows/reusable_build_maturin_linux.yml

```diff
@@ -33,9 +38,13 @@ on:
         required: false
         default: false
         type: boolean
+      python_3_13:
+        description: "Build for Python 3.13"
+        required: false
+        default: false
```

Comment:
I would suggest setting the default to true, instead of false, so we don't forget this

Reply:
This would break any package that still has pyo3 0.21.

Reply:
Yes, the corresponding .github folder would need to be updated, to ensure that the work ends up getting done